.github/workflows/publish.yaml

```diff
@@ -45,7 +45,7 @@ jobs:
       - name: Set up Nodejs 12
         uses: actions/setup-node@v1
         with:
-          node-version: 12
+          node-version: 14
```

Comment:
Create React App v5 includes loads of warnings for node 12 so I moved all to node 14.